spec/controllers/sign_up/registrations_controller_spec.rb

```diff
@@ -56,6 +56,36 @@
         )
       end
     end
+
+    context 'with thmx enabled' do
```

Comment:
I think we should stick with either the full name (threatmetrix) or the more common abbreviation (tmx).

Personally I'm not a fan of abbreviations.

```suggestion
    context 'with threatmetrix enabled' do
```